k8s/README.md

```diff
@@ -0,0 +1,185 @@
+   
+   ```bash
+   # use Spark-3.2.2 as example, it can be modified when building the docker image
+   export SPARK_HOME=<path to>/spark-3.2.2-bin-hadoop3.2
+   # set your docker image path
+   export IMAGE_NAME=<repo>:<tag>
+   # set k8s master address
+   export K8SMASTER=k8s://8440EE5F8730EDD7D8B989F704DE1DFE.gr7.us-west-2.eks.amazonaws.com:443
+   # use defualt namespace as an example
+   export SPARK_NAMESPACE=default
+   export SPARK_DRIVER_NAME=example-driver
+   
+   $SPARK_HOME/bin/spark-submit \
+        --master $K8SMASTER \
+        --deploy-mode cluster  \
+        --name example-app \
+        --driver-memory 2G \
+        --conf spark.eventLog.enabled=false \
+        --conf spark.executor.cores=1 \
+        --conf spark.executor.instances=1 \
+        --conf spark.executor.memory=4G \
+        --conf spark.executor.resource.gpu.amount=1 \
+        --conf spark.executor.resource.gpu.discoveryScript=/opt/spark/examples/src/main/scripts/getGpusResources.sh \
+        --conf spark.executor.resource.gpu.vendor=nvidia.com \
+        --conf spark.hadoop.fs.s3a.access.key=<your aws access key> \
+        --conf spark.hadoop.fs.s3a.secret.key=<your aws secret key> \
+        --conf spark.hadoop.fs.s3a.fast.upload=true \
+        --conf spark.hadoop.fs.s3a.impl=org.apache.hadoop.fs.s3a.S3AFileSystem \
+        --conf spark.kubernetes.authenticate.driver.serviceAccountName=spark \
+        --conf spark.kubernetes.container.image.pullPolicy=Always \
+        --conf spark.kubernetes.container.image=$IMAGE_NAME \
+        --conf spark.kubernetes.driver.pod.name=$SPARK_DRIVER_NAME \
+        --conf spark.kubernetes.file.upload.path=s3a://<s3 bucket to staging your jar or python script> \
+        --conf spark.kubernetes.namespace=$SPARK_NAMESPACE \
+        --conf spark.plugins=com.nvidia.spark.SQLPlugin \
+        --conf spark.rapids.memory.pinnedPool.size=2G \
+        --conf spark.task.resource.gpu.amount=1 \
+        file:///$PWD/read-s3-test.py
+   ```
+
+   After execution, check the logs of the driver pod:
+   ```bash
+   $ kubectl logs example-driver
+   ...
+   22/11/14 10:53:45 WARN RapidsPluginUtils: RAPIDS Accelerator 22.10.0 using cudf 22.10.0.
+   ...
+   22/11/14 10:54:03 WARN GpuOverrides:
+   ...
+   +-----------------+
+   |cc_call_center_id|
+   +-----------------+
+   | AAAAAAAABAAAAAAA|
+   | AAAAAAAACAAAAAAA|
+   | AAAAAAAACAAAAAAA|
+   | AAAAAAAAEAAAAAAA|
+   | AAAAAAAAEAAAAAAA|
+   | AAAAAAAAEAAAAAAA|
+   | AAAAAAAAHAAAAAAA|
+   | AAAAAAAAIAAAAAAA|
+   | AAAAAAAAIAAAAAAA|
+   | AAAAAAAAKAAAAAAA|
+   | AAAAAAAAKAAAAAAA|
+   | AAAAAAAAKAAAAAAA|
+   | AAAAAAAANAAAAAAA|
+   | AAAAAAAAOAAAAAAA|
+   | AAAAAAAAOAAAAAAA|
+   | AAAAAAAAABAAAAAA|
+   | AAAAAAAAABAAAAAA|
+   | AAAAAAAAABAAAAAA|
+   | AAAAAAAADBAAAAAA|
+   | AAAAAAAAEBAAAAAA|
+   +-----------------+
+   only showing top 20 rows
+   ```
+4. Delete the pod
+   ```bash
+   kubeclt delete pod example-driver
```

Comment:
Shall we explain how to delete the node group and the cluster by the command?

Reply:
I'm not sure, we assume the user knows how to manage their cluster or not?

Reply:
Added